node/core/pvf/common/src/executor_intf.rs

```diff
@@ -112,3 +119,248 @@ pub fn params_to_wasmtime_semantics(par: &ExecutorParams) -> Result<Semantics, S
 	sem.deterministic_stack_limit = Some(stack_limit);
 	Ok(sem)
 }
+
+#[derive(Clone)]
+pub struct Executor {
```

Comment:
Some docs would be nice. What is an `Executor`? Why do we need it? ..

Reply:
Actually it doesn't seem that useful now that the rayon threadpool has been removed. It used to look [like this](https://github.com/paritytech/polkadot/blob//node/core/pvf/worker/src/executor_intf.rs#L146). Without that, these methods can be standalone functions that take a `Config`. I won't change it here in case it could be useful again later, but I can write a ticket.

Reply:
https://github.com/paritytech/polkadot/issues/7313